env_runner.py

```diff
@@ -154,7 +154,7 @@ def restore(self):
 
   def getObjPosition(self):
     for remote in self.remotes:
-      remote.send('get_obj_position')
+      remote.send(('get_obj_position', None))
```

Comment:
Do you need this for your weird hierarchy junk?

Reply:
That's for testing. I tested the pybullet env by executing picking at the object's position. So I needed someway to access them